### PR TITLE
Test/reserve st procedure

### PR DESCRIPTION
**Pull request by victorwiebe**

### CODE REVIEW

File: src/procedures/__tests__/ReserveSecurityToken.ts [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+          code: ErrorCode.ProcedureValidationError,
+        })
+      );
+    });
+
+    test('should return an object reflecting a reserved security token', async () => {
+      const expiryDate = new Date(2035, 0);
+      const reservedDate = new Date(2020, 0);
+      const reservationObject = {
+        expiry: dateToBigNumber(expiryDate),
```
Comment (monitz87) [inline src/procedures/__tests__/ReserveSecurityToken.ts]: These should be just dates

File: src/procedures/__tests__/ReserveSecurityToken.ts [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+        reservedAt: dateToBigNumber(reservedDate),
+        ownerAddress: params.owner,
+      };
+
+      const createStub = securityTokenReservationFactoryMock.mock('create', reservationObject);
+      const findEventsStub = ImportMock.mockFunction(utilsModule, 'findEvents', [
+        {
+          args: {
+            _ticker: params.symbol,
+            _expiryDate: dateToBigNumber(expiryDate),
```
Comment (monitz87) [inline src/procedures/__tests__/ReserveSecurityToken.ts]: Use `conversionUtils` exported by `contract-wrappers`

File: src/utils/index.ts [hunk 3]
```diff
@@ -70,6 +71,10 @@ export function isValidAddress(address: string) {
   return isAddress(address);
 }
 
+export function dateToBigNumber(value: Date) {
```
Comment (monitz87) [inline src/utils/index.ts]: This shouldn't have to be redefined here

**Comment by victorwiebe** [comment]
Changes have been addressed

**Pull request merged**